regulations/tasks.py

```diff
@@ -19,11 +19,23 @@
 
 logger = get_task_logger(__name__)
 
+# The following limits are specified by the regulations.gov API
+# They are not available as a queryable endpoint
+MAX_ATTACHMENT_COUNT = 10
+VALID_ATTACHMENT_EXTENSIONS = set([
+    "bmp", "doc", "xls", "pdf", "gif", "htm", "html", "jpg", "jpeg",
+    "png", "ppt", "rtf", "sgml", "tiff", "tif", "txt", "wpd", "xml",
+    "docx", "xlsx", "pptx"])
 
-@shared_task
-def submit_comment(body):
+
+@shared_task(bind=True)
+def submit_comment(self, body):
     comment = build_comment(body)
     files = extract_files(body)
+    valid, message = validate_attachments(files)
```

Comment:
How about calling this in `regulations/views/comment:submit_comment` instead / as well? Then we could throw a 400 back to the user and not bother queuing up the submit.